ISISComputingGroup/ibex_gui
# Removed currently selected box on selection dialogs

###Description of work

Fixes https://github.com/ISISComputingGroup/IBEX/issues/1351

###To test

Open the following dialogs and confirm that the currently selected box is not present:
*Configurations->Load
*Configurations->Edit
*Components->Edit
*Synoptics->Edit

---

#### Code Review


### Functional Tests
